docs/relational-databases/performance/tune-performance-with-the-query-store.md

```diff
@@ -68,18 +68,25 @@ ORDER BY total_execution_count DESC;
 The number of queries with the longest average execution time within last hour:
```

Comment:
Nit: use of "longest average execution time" vs "highest average execution time" or "highest average duration"